Week4/MyFavoriteMovies/app/src/main/res/layout/movie_list_view_holder.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <!-- hint says to use 0dp instead of '@dimen/smallest_layout_width' for better performance
+    on layout_width -->
+
+    <TextView
+        android:id="@+id/movieTextView"
+        android:layout_width="0dp"
```

Comment:
Hmm I'm not sure what the hint is telling you, but this seems like it wouldn't work to me :/